src/app/utils/SentryUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { CommandContext } from 'slash-create';
+import * as Sentry from '@sentry/node';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export function command(target: Object, propertyKey: string, descriptor: PropertyDescriptor): any {
+	const originalValue = descriptor.value;
+	descriptor.value = async function(...args: any[]) {
+		const ctx = args[0] as CommandContext;
+
+		const transaction = Sentry.startTransaction({
+			op: 'command',
+			name: ctx.commandName,
+		});
+
+		Sentry.configureScope(scope => {
+			scope.setSpan(transaction);
+			scope.setUser({
+				id: ctx.member.user.id,
```

Comment:
@duke-eth What happens for commands that can be executed in DMs? 

As I'm enabling strict mode, it's shown that member can come back as null because the context of the execution can happen inside or outside of a guild